pkg/apis/cluster/v1alpha2/cluster_types.go

```diff
@@ -106,29 +90,52 @@ type NetworkRanges struct {
 type ClusterStatus struct {
 	// APIEndpoint represents the endpoint to communicate with the IP.
 	// +optional
-	APIEndpoints []APIEndpoint `json:"apiEndpoints,omitempty"`
+	APIEndpoint APIEndpoint `json:"apiEndpoint,omitempty"`
```

Comment:
Why move from multiple to single endpoints? 

For providers that don't have native load-balancers this would be the ideal place to retrieve a list of endpoints to load balance across (via client-side load balancing for example).

Reply:
This has been discussed in #1137 and the ultimate goal was to reduce friction.

/cc @detiber

Reply:
I don't think a change in the data model was in scope for #1137 (outside of externalizing it), nor was it highlighted in the PR summary or proposal body - This should have been a separate PR and elicited more discussion and/or highlighted as a change in the proposal body.

Reply:
Can you open an issue to follow-up? Or PR to the current proposal? The change is small enough to be a follow-up to this PR